### PR TITLE
Update RELEASE NOTES to 1.0.2

### PR DESCRIPTION
**Pull request by filbertkm**
* adapt to changes in DataValuesJavaScript (Remove mw.ext.dataValues)

### CODE REVIEW

**Comment by thiemowmde** [comment]
The removed module was replaced with two modules that already existed in DataValues JavaScript 0.6. Is there a problem with using `~0.6.0|~0.7.0` and keeping the 1.0.2 version number?

File: README.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ https://commons.wikimedia.org/wiki/File:Wikibase_JavaScript_Data_Model_1.0.svg
 
 ## Release notes
 
-### 1.0.2 (dev)
+### 1.1.0 (dev)
 
 #### Enhancements
+* Adapt to DataValuesJavaScript 0.7.0.
```
Comment (thiemowmde) [inline README.md]: *"Made compatible with DataValues JavaScript 0.7.0.*

File: composer.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	"name": "wikibase/data-model-javascript",
 	"description": "Wikibase datamodel implementation in JavaScript",
 	"require": {
-		"data-values/javascript": "~0.6.0"
+		"data-values/javascript": "~0.7.0"
```
Comment (thiemowmde) [inline composer.json]: `~0.6.0|~0.7.0`?

Comment (filbertkm) [inline composer.json]: works for me

File: README.md [hunk 3]
```diff
@@ -5,13 +5,14 @@
 at the heart of the [Wikibase software](http://wikiba.se/).
 
 ![Wikibase JavaScript Data Model UML diagram](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c2/Wikibase_JavaScript_Data_Model_1.0.svg/600px-Wikibase_JavaScript_Data_Model_1.0.svg.png)
-https://commons.wikimedia.org/wiki/File:Wikibase_JavaScript_Data_Model_1.0.svg
+		https://commons.wikimedia.org/wiki/File:Wikibase_JavaScript_Data_Model_1.0.svg
```
Comment (thiemowmde) [inline README.md]: Uh?

**Pull request merged**